### PR TITLE
e2e/instrumentation:Use e2eskipper package

### PR DESCRIPTION
**Pull request by tanjunchen**
**What type of PR is this?**
 
/kind cleanup

**What this PR does / why we need it**:

e2e/instrumentation:Use e2eskipper package

**Which issue(s) this PR fixes**:

ref:https://github.com/kubernetes/kubernetes/issues/87047

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

File: test/e2e/instrumentation/logging/elasticsearch/kibana.go [hunk 1]
```diff
@@ -22,13 +22,14 @@ import (
...
 	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/kubernetes/test/e2e/framework"
 	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
 	e2eservice "k8s.io/kubernetes/test/e2e/framework/service"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
 	instrumentation "k8s.io/kubernetes/test/e2e/instrumentation/common"
 
 	"github.com/onsi/ginkgo"
-	"k8s.io/apimachinery/pkg/util/wait"
```
Comment (tanjunchen) [inline test/e2e/instrumentation/logging/elasticsearch/kibana.go]: the order of the guide package is generally go package、custom package、third-party package
So I adjusted the original order by the way.

File: test/e2e/instrumentation/logging/stackdriver/basic.go [hunk 2]
```diff
@@ -20,14 +20,15 @@ import (
...
+	"k8s.io/apimachinery/pkg/util/uuid"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/kubernetes/test/e2e/framework"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
 	instrumentation "k8s.io/kubernetes/test/e2e/instrumentation/common"
 	"k8s.io/kubernetes/test/e2e/instrumentation/logging/utils"
 
 	"github.com/onsi/ginkgo"
-	"k8s.io/apimachinery/pkg/util/json"
-	"k8s.io/apimachinery/pkg/util/uuid"
```
Comment (tanjunchen) [inline test/e2e/instrumentation/logging/stackdriver/basic.go]: ditto

File: test/e2e/instrumentation/logging/generic_soak.go [hunk 3]
```diff
@@ -23,13 +23,14 @@ import (
...
 	"time"
 
-	"github.com/onsi/ginkgo"
 	"k8s.io/api/core/v1"
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/framework/config"
 	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
 	instrumentation "k8s.io/kubernetes/test/e2e/instrumentation/common"
 	imageutils "k8s.io/kubernetes/test/utils/image"
+
```
Comment (haosdent) [inline test/e2e/instrumentation/logging/generic_soak.go]: Is it required to change here?

File: test/e2e/instrumentation/monitoring/metrics_grabber.go [hunk 4]
```diff
@@ -26,15 +26,15 @@ import (
 	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
 	instrumentation "k8s.io/kubernetes/test/e2e/instrumentation/common"
 
-	gin "github.com/onsi/ginkgo"
-	gom "github.com/onsi/gomega"
+	"github.com/onsi/ginkgo"
```
Comment (haosdent) [inline test/e2e/instrumentation/monitoring/metrics_grabber.go]: Is it required to change this file?

File: test/e2e/instrumentation/logging/generic_soak.go (hunk 3 above)
Comment (tanjunchen) [reply test/e2e/instrumentation/logging/generic_soak.go]: no,it is not required to change this.In my memory, the import of these two packages generally does not require aliases. so i change it. This pr refer to the 87047 issue.Thanks your review

File: test/e2e/instrumentation/monitoring/metrics_grabber.go (hunk 4 above)
Comment (tanjunchen) [reply test/e2e/instrumentation/monitoring/metrics_grabber.go]: no,it is not required to change this.In my memory, the import of these two packages generally does not require aliases. so i change it. This pr refer to the 87047 issue.Thanks your review

Comment (oomichi) [reply test/e2e/instrumentation/monitoring/metrics_grabber.go]: Please don't mix different scope in a PR, keep changes simple and clean.
That makes review hard.

If we want to do that, it is nice to create another PR only for that.

Comment (tanjunchen) [reply test/e2e/instrumentation/monitoring/metrics_grabber.go]: done. i will update it

**Comment by tanjunchen** [comment]
done

**Review by oomichi — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**